tests/Etoc/SubscriptionTest.php

```diff
@@ -177,26 +176,21 @@ public function preferencesProvider() : Traversable
         ];
         yield 'repeated' => [
             [
-                'technology',
                 'elife_newsletter',
-                'technology',
             ],
             [
-                new Technology(),
```

Comment:
This example needs to be amended because it is supposed to demonstrate that including a subscription twice can be tolerated and should only be included once once processed.